src/lib/modules/ens/embarkjs.js

```diff
@@ -221,16 +227,21 @@ __embarkENS.lookup = function (address, callback) {
     return callback(err, name);
   }
 
-  return this.ens.methods.resolver(node).call((err, resolverAddress) => {
-    if (err) {
-      return cb(err);
-    }
-    if (resolverAddress === voidAddress) {
-      return cb('Address not associated to a resolver');
-    }
-    let resolverContract = new EmbarkJS.Blockchain.Contract({abi: this.resolverInterface, address: resolverAddress, web3: web3});
-    resolverContract.methods.name(node).call(cb);
-  });
+  return this.ens.methods.resolver(node).call()
+    .then(resolverAddress => {
+      if (resolverAddress === voidAddress) {
+        return cb('Address not associated to a resolver');
```

Comment:
Any particular reason one error message was put in a constant and this one not? Not that I'd be a big fan of it, but maybe we want to stay consistent?

Reply:
I thought we used the first error message more than once. I removed the constant as it's useless for a one time use.